src/Raven.Server/Documents/Handlers/Processors/Replication/AbstractReplicationHandlerProcessorForGetActiveConnections.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System.Collections.Generic;
+using System.Net.Http;
+using JetBrains.Annotations;
+using Raven.Client.Documents.Replication;
+using Raven.Client.Http;
+using Raven.Server.Documents.Replication.Stats;
+using Raven.Server.Json;
+using Sparrow.Json;
+using Sparrow.Json.Parsing;
+using static Raven.Server.Documents.Handlers.Processors.Replication.ReplicationActiveConnectionsPreview;
+
+namespace Raven.Server.Documents.Handlers.Processors.Replication
+{
+    internal abstract class AbstractReplicationHandlerProcessorForGetActiveConnections<TRequestHandler, TOperationContext> : AbstractHandlerProxyReadProcessor<ReplicationActiveConnectionsPreview, TRequestHandler, TOperationContext>
+        where TOperationContext : JsonOperationContext
+        where TRequestHandler : AbstractDatabaseRequestHandler<TOperationContext>
+    {
+        protected AbstractReplicationHandlerProcessorForGetActiveConnections([NotNull] TRequestHandler requestHandler)
+            : base(requestHandler)
+        {
+        }
+
+        protected override RavenCommand<ReplicationActiveConnectionsPreview> CreateCommandForNode(string nodeTag) => new GetReplicationActiveConnectionsInfoCommand(nodeTag);
+    }
+
+    internal class GetReplicationActiveConnectionsInfoCommand : RavenCommand<ReplicationActiveConnectionsPreview>
```

Comment:
move this to commands in the server?

Reply:
Done on https://github.com/ravendb/ravendb/pull/14908